### ISSUE TITLE
Document commands to install latest release of kubectl

### ISSUE THREAD

**Issue by Sharaf5** [kubernetes/kubernetes#129149]
isnt there a way to install latest version rather than going through the same installation process for each new version
many poeple like to be mostly up to date

this was a good workaround but it's not working now
```
kv=$( curl -L -s https://dl.k8s.io/release/stable.txt | cut -f1,2 -d'.' )
curl -fsSL https://pkgs.k8s.io/core:/stable:/$kv/deb/Release.key | sudo gpg --dearmor -o /usr/share/keyrings/kubernetes.gpg;
echo "deb [signed-by=/usr/share/keyrings/kubernetes.gpg] https://pkgs.k8s.io/core:/stable:/${kv}/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
```
but recently we need to delete the patch number from version ( b in semantic versioning vM.m.p-build )

cant you make an alise for `latest` version in these links
```
https://pkgs.k8s.io/core:/stable:/latest/deb/Release.key
https://pkgs.k8s.io/core:/stable:/latest/deb/
```

**Comment by sftim** [comment]
/retitle Make it easy to install latest release of `kubectl` via scripts

**Comment by saschagrunert** [comment]
cc @kubernetes/release-managers 

I could imagine that we populate a `latest` project through krel when we have all information at hand during release build.

Other than that I don't see many ways for a simple alias, because we require the information what the latest stable is as outlined in the workaround (first comment).

**Comment by sftim** [comment]
I wouldn't prioritize this work. The people writing scripts might have to write 5 lines rather than 2, but that's an outcome I think we can bear.

**Comment by Sharaf5** [comment]
> I could imagine that we populate a `latest` project through krel when we have all information at hand during release build.

and the process but I thought making an aliese for url is easy to do
anyway you how perioritize your work so I'm so grateful for your time to discus, so in case of you changed your minds :

there is two easy way to do that depending on your server (which I dont know) :
1. If the server serves from files like php : just make a symbolic link that is updated so the server will find it if it support symbolic links
2. I your server serves from a pre-defined routes python or node : a route could be built for latest and a copy of the latest version keys and data are there, or just a file or a db record containing the latest stable build number to redirect to, so one dont need to repeat installation process intirly during update

symbolic links / copied dirs and files / version file / db record : could be updated at the end of successful release

Of course you know your servers more than me and you know what to perioritize

**Comment by xmudrii** [comment]
This is actually a k/release issue, I'll transfer it there so we have a track of it.
/transfer release

/triage accepted
/remove-sig cli
/lifecycle frozen

This can be done on the OBS side by using [the aggregate function](https://openbuildservice.org/help/manuals/obs-user-guide/cha-obs-basicworkflow#sec-obs-basicworkflow-reuse). We in fact use that quite extensively already, but `krel` doesn't support it so we're always doing it manually. This means we would first have to extend `krel` to support aggregating packages before we can tackle this issue.

Once that's done, we could have a project like `pkgs.k8s.io/core:/latest:/cli-tools` where we aggregate `kubectl` packages.

However, as other folks said, this is not a priority for us at the moment.
